tests/namespaces.js

```diff
@@ -0,0 +1,3 @@
+import { Namespace } from "rdflib";
+
+export const EXT = new Namespace("http://mu.semte.ch/vocabularies/ext/");
```

Comment:
We don't export this namespace from the [namespaces.js](https://github.com/lblod/submission-form-helpers/blob/eef665113699c80a390e82670595e3efd2c38a68/src/namespaces.js) file yet but I wasn't sure if we should, so added it to the tests folder instead.

Should it be a public export?

Reply:
thinking: not sure... ext feels like a bit of a 'temp, not official' namespace, so it could be strange if we were to export it, making it feel more official

Reply:
Yea, that was my feeling as well. I'll keep it in the tests folder, thanks!